.github/workflows/main.yml

```diff
@@ -4,26 +4,39 @@ on:
   workflow_dispatch:
 
 jobs:
-  build:
-    name: build
+  ubuntu:
+    name: Ubuntu
     runs-on: ubuntu-latest
-
     steps:
-      - uses: actions/checkout@v4
-
+      - name: Check out Git repository
```

Comment:
No need to add a name here, could you revert this?

Reply:
Yes, no need for an name. I have reverted this.